mmedit/models/editors/restormer/restormer_net.py

```diff
@@ -0,0 +1,385 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+import numbers
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from einops import rearrange
+from mmengine.model import BaseModule
+
+from mmedit.registry import MODELS
+
+
+def to_3d(x):
+    return rearrange(x, 'b c h w -> b (h w) c')
+
+
+def to_4d(x, h, w):
+    return rearrange(x, 'b (h w) c -> b c h w', h=h, w=w)
+
+
+class BiasFree_LayerNorm(BaseModule):
```

Comment:
Add docstrings for every class and every forward function. You can reference other models.